core/src/main/scala/com/salesforce/op/evaluators/EvaluationMetrics.scala

```diff
@@ -190,6 +190,8 @@ sealed abstract class ForecastEvalMetric
 object ForecastEvalMetrics extends Enum[ForecastEvalMetric] {
   val values: Seq[ForecastEvalMetric] = findValues
   case object SMAPE extends ForecastEvalMetric("smape", "symmetric mean absolute percentage error")
+  case object MASE extends ForecastEvalMetric("mase", "mean absolute scaled error ")
+  case object seasonalError extends ForecastEvalMetric("seasonalError", "seasonal error")
```

Comment:
CamelCase the object name to `case object SeasonalError`

Reply:
It seems that if I camel case that object then I will have to camel case also field name in `ForecastMetrics` - is that expected?

Reply:
Hmm, no. you can show me tomorrow perhaps? 

The following should work just fine
```scala
case object SeasonalError extends ForecastEvalMetric("seasonalError", "seasonal error")
```